test/unicode/utf8.jl

```diff
@@ -1,17 +1,5 @@
 # This file is a part of Julia. License is MIT: http://julialang.org/license
 
-## Test for CESU-8 sequences
-
-let ch = 0x10000
-    for hichar = 0xd800:0xdbff
-        for lochar = 0xdc00:0xdfff
-            @test convert(String, String(Char[hichar, lochar]).data) == string(Char(ch))
```

Comment:
Yeah, I should probably just add that one back. It was defined as a convert method, along with other questionable conversions like `convert(::Type{String}, ::Vector{Int32})`, which simply shouldn't exist.